Project-3/swatipriya_project3/ReadMe.md

```diff
@@ -0,0 +1,74 @@
+PROJECT NAME - WEATHER 
```

Comment:
Please add a link to your website and optionally screenshots of the functionality here itself.

Thanks!

Reply:
Successfully added!!